File: applicatie/src/main/java/com/vdzon/administratie/bankimport/regelParsers/DefaultRegelParser.java
package com.vdzon.administratie.bankimport.regelParsers;

public class DefaultRegelParser implements RegelParser {

    @Override
    public boolean match(String omschrijving) {
        return false;
    }


    @Override
    public String extractNaam(String omschrijving) {
        return "Onbekend";
    }

    @Override
    public String extractOmschrijving(String omschrijving) {
        return "Onbekend";
    }
}

File: applicatie/src/main/java/com/vdzon/administratie/bankimport/regelParsers/DefaultRegelParser.kt
package com.vdzon.administratie.bankimport.regelParsers

class DefaultRegelParser : RegelParser {

    override fun match(omschrijving: String): Boolean {
        return false
    }


    override fun extractNaam(omschrijving: String): String {
        return "Onbekend"
    }

    override fun extractOmschrijving(omschrijving: String): String {
        return "Onbekend"
    }
}
